### ISSUE TITLE
Support more file views like list and content

### ISSUE THREAD

**Issue by emmatyping** [IronManMark20/Explore10#35]

**Comment by Krutonium** [comment]
Perhaps an image view as well, like explorer in Windows XP did for My Pictures.

**Comment by emmatyping** [comment]
@PFCKrutonium I am not sure what your talking about. I haven't used XP in forever,  can you post a picture of what you  mean?

**Comment by Krutonium** [comment]
![winxp_rc_gallery_03 1](https://cloud.githubusercontent.com/assets/3945538/11050189/e5dd1304-8711-11e5-85ef-e3dccf1c717a.gif)

**Comment by emmatyping** [comment]
Ah, yeah that is nice. I mean, I guess that is kinda duplicative with a picture viewer, but sure, we can add that.

**Comment by Krutonium** [comment]
I agree it is, but it allows you to rotate the images and get a better look without having to open another program.

I kinda miss this in newer versions of Windows.
